versions/3.1.0.md

```diff
@@ -2276,11 +2276,16 @@ This object is an extended subset of the [JSON Schema Specification Wright Draft
 For more information about the properties, see [JSON Schema Core](https://tools.ietf.org/html/draft-wright-json-schema-00) and [JSON Schema Validation](https://tools.ietf.org/html/draft-wright-json-schema-validation-00).
 Unless stated otherwise, the property definitions follow the JSON Schema.
 
-##### Properties 
+##### Properties
```

Comment:
For the record: just notice that the paragraph above still says the Schema object is an extended subset of Draft-00.

Reply:
Ooooo nice catch, thank you. I'll wait for draft next then we're all set.

Reply:
Actually this is what the WIP links are for! Updated, please take a look.